### ISSUE TITLE
[bug] `rh-tooltip` should announce content to screen readers on focus

### ISSUE THREAD

**Issue by hellogreg** [RedHat-UX/red-hat-design-system#2051]
`rh-tooltip` currently only presents its contents visually upon hover and focus.

As demonstrated at [its docs demo page](https://ux.redhat.com/elements/tooltip/demo/), tooltips are keyboard-focusable, by default. And they do display their text content visually upon focus. However, this content is not currently read by screen readers.

![Screencap of a focused tooltip](https://github.com/user-attachments/assets/8ef248b1-612e-40cf-b5cb-91bf735be9dd)
